src/types/inc/utils.hpp

```diff
@@ -19,8 +19,10 @@ namespace Microsoft::Console::Utils
     GUID GuidFromString(const std::wstring wstr);
     GUID CreateGuid();
 
-    std::wstring ColorToHexString(const COLORREF color);
-    COLORREF ColorFromHexString(const std::wstring wstr);
+    std::wstring ColorToHexStringW(const COLORREF color);

```

Comment:
aaaaaaaa is this better than `ColorToHexString<wstring>`

Reply:
It's unused now, lets just pull it entirely